src/main/java/fr/iut/pathpilotapi/client/ClientRestController.java

```diff
@@ -98,7 +99,22 @@ public ResponseEntity<Client> deleteClient(
             @RequestBody ClientDeleteRequestModel clientDeleteRequestModel
     ) {
         Salesman salesman = (Salesman) authentication.getPrincipal();
-        Client client = clientService.getClientById(clientDeleteRequestModel.getId());
+        return getResponseEntityDeleteClient(clientDeleteRequestModel.getId(), salesman);
+    }
+
```

Comment:
add swagger doc there

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-API/pull/9/commits/fdd235c334a232cc8c6e630ce6293edfef524cd6